stl/inc/chrono

```diff
@@ -612,14 +612,13 @@ namespace chrono {
         _NODISCARD static time_point now() noexcept { // get current time
 #if (defined(_M_IX86) || defined(_M_X64) || defined(_M_ARM) || defined(_M_ARM64)) && !defined(_M_CEE_PURE)
             // Implement atomics avoiding <atomic> header dependency
-            static struct {
-                volatile long long _Freq        = LLONG_MAX;
-                volatile long long _Ctr_base    = LLONG_MAX;
-                volatile long long _Result_base = LLONG_MAX;
-            } _Cached;
-            const long long _Freq_from_cache = _Atomic_load_ll_relaxed(&_Cached._Freq);
-            const long long _Ctr_base        = _Atomic_load_ll_relaxed(&_Cached._Ctr_base);
-            const long long _Result_base     = _Atomic_load_ll_relaxed(&_Cached._Result_base);
+            static volatile long long _Cached_Freq        = LLONG_MAX;
+            static volatile long long _Cached_Ctr_base    = LLONG_MAX;
+            static volatile long long _Cached_Result_base = LLONG_MAX;

```

Comment:
Naming nitpick: These should be `_Ugly_snake_case`, not `_Ugly_Camel_snake_chimera`.